montecristo/src/main/kotlin/com/datastax/montecristo/model/application/CassandraYaml.kt

```diff
@@ -31,7 +31,9 @@ data class CassandraYaml(val data : JsonNode)  : YamlConfig(data) {
     val partitioner get() = get("partitioner")
     val memtableAllocationType get() = get("memtable_allocation_type")
     val authorizer get() = get("authorizer")
-    private val rowCacheSizeInMB get() = (get("row_cache_size","0")).toInt()
+    val rowCacheSizeInMB get() = (
+            get("row_cache_size", "0").replace("MiB", "")
```

Comment:
could be GiB (or KiB), or GB, MB, KB

Reply:
Could I then delete all non-numeric characters ? Something like `.replace("\D+", "")` ?

Reply:
yeah, i agree to that

Reply:
Pushed a commit doing this.